src/components/SeparatorVirtualGrid.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+	<h2 class="grid-title" :style="{height: item.height + 'px'}">
+		{{ item.injected.month }}
+		<span>{{ item.injected.year }}</span>
+	</h2>
+</template>
+
+<script>
+export default {
+	name: 'SeparatorVirtualGrid',
+
+	inheritAttrs: false,
+
+	props: {
+		item: {
+			type: Object,
+			required: true,
+			injected: {
```

Comment:
Does that exists? Never saw it before: `injected`
https://vuejs.org/v2/api/#props

Reply:
it's a custom property from my lib

Reply:
But it's not passed. you cannot nest props in Vue.
If you needed to pass a default, then you need to use the default key, `injected` means nothing to vue here. :)

Reply:
ah right, i think i'm mixing things up with Typescript, you're right i only need to have the root property and i can still access to item.injected.xxxx

Reply:
This is fixed